src/viam/services/vision/client.py

```diff
@@ -127,7 +135,13 @@ async def get_object_point_clouds(
         response: GetObjectPointCloudsResponse = await self.client.GetObjectPointClouds(request, timeout=timeout)
         return list(response.objects)
 
-    async def do_command(self, command: Mapping[str, ValueTypes], *, timeout: Optional[float] = None) -> Mapping[str, ValueTypes]:
+    async def do_command(
+        self,
+        command: Mapping[str, ValueTypes],
+        *,
+        timeout: Optional[float] = None,
+        **__,
```

Comment:
(q) Why is it that this is necessary for `do_command` but not, e.g., `get_object_point_clouds` above?

Reply:
Because the abstract super method for `get_object_point_clouds` doesn't include `**kwargs` in the input list, whereas `do_command` does